src/handlers/poster/gewis/components/GewisProgressBar.tsx

```diff
@@ -1,5 +1,5 @@
-import Clock from './Clock';
-import ProgressBarSlider from './ProgressBarSlider';
+import ProgressBarSlider from '../../components/ProgressBarSlider.tsx';
+import Clock from './Clock.tsx';
```

Comment:
Why do these imports have a file extension?

Reply:
Honestly, I never import, WebStorm does it for me, gotta change some settings I think :sweat_smile: